test/DebugInfo/NonSemantic/Shader200/DebugLinePriority.spt

```diff
@@ -1,14 +1,15 @@
 ; RUN: llvm-spirv -to-binary %s -o %t.spv
+; RUN: spirv-val %t.spv
```

Comment:
Interesting, I haven't yet added NonSemantic.Shader.DebugInfo.200 to spirv-val and spirv-headers, does it mean, that spir-v val skips nonsemantic instruction sets?

Reply:
I suppose so; I haven't looked in detail in the spirv-val sources, but it seems to exclude NonSemantic instructions in various places by checking for a `NonSemantic.` prefix.  The best "proof" I have is that this test passes `spirv-val` after fixing the issues. :-)